practice.md

```diff
@@ -29,8 +29,23 @@ As [the Free Software Foundation says](https://www.gnu.org/licenses/gpl-faq.html
 
 > Yes. If the improvements are written by US government employees in the course of their employment, then the improvements are in the public domain. However, the improved version, as a whole, is still covered by the GNU GPL. There is no problem in this situation.
 
+### Contributor License Agreements (CLAs)
 
-#### How to license 18F repos
+Some external projects have CLAs. You cannot sign these yourself, in your official capacity.
+
+1. See if there is an organizational CLA available.
```

Comment:
How do we see this, besides the list below?  Are the channels to ask in, or is this the canonical spot?